server/src/test/java/org/apache/druid/server/QuerySchedulerTest.java

```diff
@@ -399,6 +399,49 @@ public void testMisConfigHiLo()
   }
 
 
+  @Test
+  public void testConfigManual()
+  {
+    final Injector injector = createInjector();
+    final String propertyPrefix = "druid.query.scheduler";
+    final JsonConfigProvider<QuerySchedulerProvider> provider = JsonConfigProvider.of(
+        propertyPrefix,
+        QuerySchedulerProvider.class
+    );
+    final Properties properties = new Properties();
+    properties.put(propertyPrefix + ".numThreads", "10");
+    properties.put(propertyPrefix + ".laning.strategy", "manual");
+    properties.put(propertyPrefix + ".laning.lanes.one", "1");
+    properties.put(propertyPrefix + ".laning.lanes.two", "2");
+    provider.inject(properties, injector.getInstance(JsonConfigurator.class));
+    final QueryScheduler scheduler = provider.get().get().get();
+    Assert.assertEquals(10, scheduler.getTotalAvailableCapacity());
+    Assert.assertEquals(1, scheduler.getLaneAvailableCapacity("one"));
+    Assert.assertEquals(2, scheduler.getLaneAvailableCapacity("two"));
```

Comment:
maybe also test negative case (like lane name "three") and see we get NO_CAPACITY?

Reply:
This isn't actually true currently, non-existent lanes are currently treated the same as not having a lane. I did this in  #9407 because it seemed friendlier, but an argument can probably made for making undefined lanes that are present be treated as having 0 capacity which would make what you are suggesting true.

Reply:
Oh, I maybe interpreted this as something deeper, suggesting that we should treat queries that specify non-existent lanes as a 0 capacity lane, because of my poorly named `NO_CAPACITY` variable which I am going to rename as `UNAVAILABLE` pending this discussion. If you're just suggesting that we check that non-existent lanes return `-1` I can add that test.

Reply:
Im just suggesting to add test that check that non-existent lanes return -1

Reply:
added some asserts to existing tests to show that non-existent lane is unavailable